src/js/api.js

```diff
@@ -0,0 +1,31 @@
+export async function getPokemons() {
+    const response = await fetch("https://pokeapi.co/api/v2/pokemon");
+    const pokemons = await response.json();
+
+    return pokemons;
+}
+
+export async function getPageOfPokemons(page) {
+    const response = await fetch(`https://pokeapi.co/api/v2/pokemon?offset=${(page - 1) * 20}&limit=20`);
+    const pokemons = await response.json();
+
+    return pokemons;
+}
+
+export async function getPokemon(ID) {
```

Comment:
`(id)` without uppercase, please.

Reply:
I leave the uppercase for the constants from now on